pallets/liquidity-pools/src/defensive_weights.rs

```diff
@@ -19,122 +19,94 @@ pub trait WeightInfo {
 	fn update_member() -> Weight;
 	fn transfer() -> Weight;
 	fn set_domain_router() -> Weight;
+	fn add_currency() -> Weight;
+	fn allow_investment_currency() -> Weight;
+	fn disallow_investment_currency() -> Weight;
 	fn schedule_upgrade() -> Weight;
 	fn cancel_upgrade() -> Weight;
 	fn update_tranche_token_metadata() -> Weight;
+	fn freeze_investor() -> Weight;
+	fn unfreeze_investor() -> Weight;
 }
 
 // NOTE: We use temporary weights here. `execute_epoch` is by far our heaviest
 //       extrinsic. N denotes the number of tranches. 4 is quite heavy and
 //       should be enough.
 const N: u64 = 4;
 
+/// NOTE: Defensive hardcoded weight taken from pool_system::execute_epoch. Will
+/// be replaced with real benchmark soon.
+
+fn default_defensive_weight() -> Weight {
+	Weight::from_parts(124_979_771, 19974)
+		.saturating_add(Weight::from_parts(58_136_652, 0).saturating_mul(N))
+		.saturating_add(RocksDbWeight::get().reads(8))
+		.saturating_add(RocksDbWeight::get().reads((7_u64).saturating_mul(N)))
+		.saturating_add(RocksDbWeight::get().writes(8))
+		.saturating_add(RocksDbWeight::get().writes((6_u64).saturating_mul(N)))
+		.saturating_add(Weight::from_parts(0, 17774).saturating_mul(N))
+}
```

Comment:
Thanks for factoring this! ❤️ 

Could we even group these weights or use a constant as in the router part? Probably with a lower because it just does some action + enqueuing a message.